torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -115,9 +118,10 @@ bool removeExtraNodeOutputs(Node *n) {
 
 void ConstantPropagation(Node* n, bool recurse) {
   bool constant_inputs = (n->inputs().size() > 0) &&
-    std::all_of(n->inputs().begin(), n->inputs().end(), [&](Value* v) {
-      return v->node()->kind() == prim::Constant;
-    });
+      std::all_of(n->inputs().begin(), n->inputs().end(), [&](Value* v) {
+                           return v->node()->kind() == prim::Constant &&
+                               v->node()->output()->type() != WorldType::get();
```

Comment:
This is gross, but we only need this until we can pass World tokens in and out of graphs